github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-periodics.yaml

```diff
@@ -126,9 +171,26 @@ periodics:
       - "/bin/bash"
       - "-c"
       - >
+        # For getting centos image tag from amd64 prow job and use same for xs390x and manifest-list images
+        SHORT_SHA=$(git rev-parse --short HEAD) &&
+        GCS_FILE_PATH=release/kubevirt/kubevirtci/amd64-centos9-$SHORT_SHA &&
+        CHECK_INTERVAL=30 &&
+        source /usr/local/bin/gcs_restapi.sh &&
+        while true; do
+            if stat_gcs_file kubevirt-prow "$GCS_FILE_PATH"; then
+                echo "File $GCS_FILE_PATH is now available."
+                break
+            else
+                echo "File $GCS_FILE_PATH not found. Checking again in $CHECK_INTERVAL seconds."
+                sleep $CHECK_INTERVAL
+            fi
+        done &&
+        export KUBEVIRTCI_TAG=$(cat_gcs_file kubevirt-prow "$GCS_FILE_PATH") || { echo "Failed to fetch KUBEVIRTCI_TAG"; exit 1; } &&
+        echo "Fetched KUBEVIRTCI_TAG: $KUBEVIRTCI_TAG" &&
         cat $QUAY_PASSWORD | podman login --username $(<$QUAY_USER) --password-stdin quay.io &&
         ./hack/bump-centos-version.sh &&
         GIT_ASKPASS=../project-infra/hack/git-askpass.sh ../project-infra/hack/git-pr.sh -c "PHASES=linux BYPASS_PMAN_CHANGE_CHECK=true ./publish.sh" -r kubevirtci -b bump-centos-stream -T main -p $(pwd) -s "Automatic bump of CentOS Stream to latest"
```

Comment:
I think I would prefer the creation of the PR to remain on the x86 job - the s390x will eventually become consistent.

Reply:
> @brianmcarey I understand your concern when s390x job fails.
> 
>     * If we move PR creation to x86, then
> 
>     * [ ]  s390x jobs which consume this image will fail, as image mentioned in cluster-provision/k8s/base-image doesn't exist for s390x.

This is probably acceptable for the short period of time it will take s390x to be consistent. 
> 
>     * [x]  x86 jobs which consume the image will get the latest.
> 
>     * If we keep PR creation in s390x, then
> 
>     * [x]  s390x jobs which consume this image will pass, but using "existing" version.
> 
>     * [ ]  x86 jobs which consume the image will pass, but using "existing" version. If we can't wait till s390x jobs is fixed and we want to proceed with using latest published x86 image (which is pushed already independent of PR creation with same tag as multi-arch image would get), then it needs manual PR creation instead of by kubevirt-bot, updating centos version and image tag from prow job's output.
> 
> 
> So I think this is more of a decision we've to make depending on what is important to us, s390x builds also passing with older version, till the issue is fixed or importance/urgency of having latest centos in x86 builds. As you know better, I will do changes as per your decision. Please let me know, Thanks!

From my point of view - I want to take the least intrusive path to the x86 flow. We can make follow-up changes to improve the integration once we have verified the stability of the s390x publishing etc. but for now I would prefer for s390x to take the approach of being eventually consistent with x86. 

If we see that the s390x publish is very stable and never fails then we can improve the integration.

Reply:
> This is probably acceptable for the short period of time it will take s390x to be consistent.

Yes, in the case when "periodic-kubevirtci-bump-centos-base-s390x" job eventually passes. But if "periodic-kubevirtci-bump-centos-base-s390x" job fails, the "publish-kubevirtci-s390x" job will start failing till 'periodic-kubevirtci-bump-centos-base-s390x job is fixed and next build is promoted' or 'CentOS bump PR reverted to old build, where both arch centos base images (+multi-arch manifest) were published'.

Similarly when "publish-kubevirtci-s390x" fails, it affects jobs using 'make cluster up'.

But still as we discussed, probably this is fine as at least this wouldn't affect jobs of kubevirt/kubevirt (because it has its own reference of kubevirtci provider version) or other repos, but just affect dependent jobs on kubevirtci repo only. **Is my understanding correct?**

With this understanding and as per our last discussion, I've moved PR creation and commiting latest file in GCS to respective x86 jobs.